scripts/package_chrome_extension.sh

```diff
@@ -18,7 +18,7 @@ cd ..
 if [ "${TAG}zz" == "zz" ]; then
     # Package the extension with Chromium, if we're not packaging a public version
     echo "Signing the extension for Chrome with a local Chromium, version $VERSION"
-    chromium-browser --no-sandbox --pack-extension=tmp --pack-extension-key=./scripts/kiwix-html5.pem
+    chromium-browser --no-sandbox --pack-extension=tmp --pack-extension-key=./scripts/kiwix-js.pem
```

Comment:
This one will be more complicated to change, because it involves a "secret" file (kiwix-html5.pem), which is not commited, and included (for Travis) in an encrypted form inside secret_files.tar.gz.enc.